pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -153,6 +157,7 @@ def get_unit_data(self, unit):
             'email_md5': None,
             'source': unit.source,
             'target': unit.target,
+            'submitted_on': make_aware(datetime.now()),
```

Comment:
This would be a valid *import time*, however as I understand it, we are looking for *translation time*. When importing from files, we effectively don't know when a unit was translated, so I think this line is wrong and would result misleading. Thoughts @translate/core?